presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParser.java

```diff
@@ -1623,30 +1624,38 @@ public void testGrant()
         assertStatement("GRANT INSERT, DELETE ON t TO u",
                 new Grant(
                         Optional.of(ImmutableList.of("INSERT", "DELETE")),
-                        false,
+                        Optional.empty(),
                         QualifiedName.of("t"),
                         new PrincipalSpecification(PrincipalSpecification.Type.UNSPECIFIED, new Identifier("u")),
                         false));
         assertStatement("GRANT SELECT ON t TO ROLE PUBLIC WITH GRANT OPTION",
                 new Grant(
                         Optional.of(ImmutableList.of("SELECT")),
-                        false, QualifiedName.of("t"),
+                        Optional.empty(),
+                        QualifiedName.of("t"),
                         new PrincipalSpecification(PrincipalSpecification.Type.ROLE, new Identifier("PUBLIC")),
                         true));
-        assertStatement("GRANT ALL PRIVILEGES ON t TO USER u",
+        assertStatement("GRANT ALL PRIVILEGES ON TABLE t TO USER u",
```

Comment:
separate commit, you are changing not related test. As I understand this test case with present `TABLE` keyword in GRANT statement was missing, right?

Reply:
Correct